src/ReleaseHistory.md

```diff
@@ -1,5 +1,33 @@
 # SARIF Viewer Visual Studio extension Release History
 
+## To Be Released
+## [SARIF Viewer](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer)
+* BUGFIX: Error list expando for long message does not work.
+* FEATURE: Generate Popup from Xaml.
+
+## [SARIFER](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifer)
+* Publish official release to marketplace.
+* Enable Spam Analyzer Dynamic Mode.
```

Comment:
SARIFER appears to have been released as v 3.0.73.27569. Could you please add the version number?

Reply:
Will update. I don't see this release at [https://github.com/microsoft/sarif-visualstudio-extension/releases](https://github.com/microsoft/sarif-visualstudio-extension/releases) where did you see it?

Reply:
The VS marketplace.